src/django/api/migrations/0130_introduce_separate_data_gathering_functions_for_the_index_table_columns.py

```diff
@@ -11,6 +11,7 @@
 def introduce_separate_data_gathering_functions(apps, schema_editor):
     helper.run_sql_files([
         '0130_rename_facilityindexnew_forward.sql',
+        '0130_remove_ppe_fields_from_facilityindex.sql',
```

Comment:
There should be a separate migration(new one) for removing the ppe fileds. We can't remigrate to 0130 migration.